packages/styled-system/src/responsive/Breakpoints.ts

```diff
@@ -0,0 +1,11 @@
+// Those breakpoints could be in the token solution, but we decided to keep them here for now, since there is no way to
+// use breakpoint tokens in css media queries.
+export const Breakpoints = {
```

Comment:
Where do the values below come from?

Reply:
I believe those are Tailwind CSS breakpoints. 

I'm thinking more and more that we should have design tokens representing those values, so we could document them properly.

Reply:
I should have made it clear that I was talking about breakpoint values. If you look at the Figma, the values don't correspond to the defined size. 

**Example**
<img width="592" alt="Capture d’écran, le 2023-11-13 à 10 08 56" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/d1eaadb6-7258-45bb-a327-17a8e675863a">

**Source**
https://www.figma.com/file/XCmj4GG4bdntwMgAa4Hmtv/%F0%9F%90%B8-Hopper-Styles?type=design&node-id=787%3A1280&mode=design&t=J77bp9Czf5Yhyzwc-1

Reply:
My answers wasn't good either, i meant we stole the values from tailwindcss, not just the name haha. 

The figma's doc seems "broken", with breakpoints not available for values between 769 and 991, and xl being included in the lg section. I asked francis T to talk with Francis B. I will update the value once everyting is fixed. Good catch thank you